binaries/download_binaries.py

```diff
@@ -0,0 +1,151 @@
+    # Set executable permissions for the downloaded binaries.
+    default_permissions = 0o755
+    os.chmod(file_path, default_permissions)
+    # Add the this file we just downloaded to the downloaded files.
+    downloaded_files.append(file_name)
+
+  assets: List[dict] = []
+  for end_point in github_api_end_points:
+    res = requests.get(end_point,
+                       # This header is recommended by github docs.
+                       headers={'Accept': 'application/vnd.github.v3+json'})
+    res.raise_for_status()
+    res_in_json = res.json()
+    # res_in_json[0] will grab the latest release.
+    assets.extend(res_in_json[0]['assets'])
+
+  download_threads: List[Thread] = []
+  downloaded_files: List[str] = []
+
+  # Loop over all the assets and download them.
+  for asset in assets:
+    download_url: str = asset['browser_download_url']
+    download_threads.append(Thread(target=download_binary,
+                                   args=(download_url, downloaded_files)))
+    # Start downloading the asset.
+    download_threads[-1].start()
+
+  # Wait until all the binaries are downloaded.
+  while(any(download_thread.is_alive() for
+            download_thread in download_threads)):
+    download_threads = [download_thread for download_thread in download_threads
+                        if download_thread.is_alive()]
+    time.sleep(1)
+
+  return downloaded_files
+
+
+def select_binaries(binaries_names, name_contains: List[str]) -> List[str]:
+  """A non-reliable method to automate selecting which binaries goes
+  for which platform."""
+
+  # The subset of the binaries we will select.
+  selected_binaries: List[str] = []
+  for binary_name in binaries_names:
+    # If we have every substring the `binary_name` so we can select it safely.
+    if all((substring in binary_name) for substring in name_contains):
+      selected_binaries.append(binary_name)
+  return selected_binaries
+
+
+def main():
+  # https://api.github.com/repos/{owner}/{repo}/releases
+  releases_urls = ['https://api.github.com/repos/joeyparrish/static-ffmpeg-binaries/releases',
+                   'https://api.github.com/repos/google/shaka-packager/releases']
+
+  # Download file whose names starting with one of these strings.
+  starts_with = ['ffmpeg', 'ffprobe', 'packager']
+  binaries_names = download_github_release_binaries(streamer_binaries.__name__,
+                                                    releases_urls,
+                                                    starts_with)
+
+  # Sort the binaries based on platform.
+  package_data = {
+      # 64-bit Windows
+      'win_amd64': select_binaries(binaries_names,
+                                   name_contains=['win',]),
+      # 64-bit Linux
+      'manylinux1_x86_64': select_binaries(binaries_names,
+                                           name_contains=['linux', 'x64',]),
+      # Linux on ARM
+      'manylinux2014_aarch64': select_binaries(binaries_names,
```

Comment:
@joeyparrish As you see, it is not so reliable way to produce the yaml file we use for building the wheels xD.
Would you please name the new upcoming ARM static build with a name that has `arm64` in it, or maybe `aarch64`, and have an `x64` for linux then so we can differentiate between the two linux es.
We can also do the same for windows and macos, just for clarity.
`packager-win-x64.exe`, `packager-osx-x64.dmg` something like this.

Reply:
Totally.  I will update Packager's workflows to match the naming scheme I used for FFmpeg.